hosted/gke/support_matrix/support_matrix_suite_test.go

```diff
@@ -36,12 +36,17 @@ var (
 func TestSupportMatrix(t *testing.T) {
 	RegisterFailHandler(Fail)
 	var err error
-	ctx = helpers.CommonBeforeSuite(helpers.Provider)
+	ctx = helpers.CommonBeforeSuite()
 	availableVersionList, err = helper.ListSingleVariantGKEAvailableVersions(ctx.RancherClient, project, ctx.CloudCred.ID, zone, "")
 	Expect(err).To(BeNil())
 	RunSpecs(t, "SupportMatrix Suite")
 }
 
+var _ = SynchronizedBeforeSuite(func() []byte {
+	helpers.CommonSynchronizedBeforeSuite()
+	return nil
+}, func() {})
+
```

Comment:
ctx is initialized before SynchronizedBeforeSuite() can run so it panics while initializing a rancher client. To be fixed.